tests/integration/aof.tcl

```diff
@@ -478,22 +478,22 @@ tags {"aof external:skip"} {
     }
 
     test {EVAL timeout with slow verbatim Lua script from AOF} {
-        create_aof $aof_dirpath $aof_file {
-            append_to_aof [formatCommand select 9]
-            append_to_aof [formatCommand eval {redis.call('set',KEYS[1],'y'); for i=1,1500000 do redis.call('ping') end return 'ok'} 1 x]
-        }
-
-        start_server [list overrides [list dir $server_path appendonly no lua-time-limit 1 aof-use-rdb-preamble no]] {
+        start_server [list overrides [list dir $server_path appendonly yes lua-time-limit 1 aof-use-rdb-preamble no]] {
             # generate a long running script that is propagated to the AOF as script
             # make sure that the script times out during loading
+            create_aof $aof_dirpath $aof_file {
+                append_to_aof [formatCommand select 9]
+                append_to_aof [formatCommand eval {redis.call('set',KEYS[1],'y'); for i=1,1500000 do redis.call('ping') end return 'ok'} 1 x]
+            }
             set rd [redis_deferring_client]
-            r config set appendonly yes
             set start [clock clicks -milliseconds]
             $rd debug loadaof
             $rd flush
-            after 100
-            catch {r ping} err
-            assert_match {LOADING*} $err
+            wait_for_condition 1000 100 {
```

Comment:
This will wait up to 100s, is it too long?

Reply:
i'm not sure i mind the long delay (probably 20 seconds is enough, but as long as it normally exists fast, i care less about how long it takes to detect the error).

we do however have `wait_for_log_messages`, let's use it rather than a look that uses `count_log_message`.
also, like the original code, i still want to assert that PING returns a LOADING error.

Reply:
changed to wait_for_log_messages and wait 10S